core/MyTransport.cpp

```diff
@@ -825,7 +826,15 @@ void transportProcessMessage(void)
 #endif //defined(MY_OTA_LOG_RECEIVER_FEATURE)
 #if defined(MY_GATEWAY_FEATURE)
 		// Hand over message to controller
-		(void)gatewayTransportSend(_msg);
+		if (_msg.getType() == V_MULTI_MESSAGE) {
```

Comment:
I think you should do the same multi message handling below in (original and unchanged) line 832: `receive(_msg);`.
MySensors allows also communication from node to node, and this way you would also support multi message for this N2N use case.

Reply:
Precious info! I have updated it now.